tests/test_servicebus.py

```diff
@@ -135,14 +147,53 @@ def _generate_servicebus_metadata(self):
             'application/json', 'string'
         )
         mocked_metadata['SequenceNumber'] = meta.Datum(3, 'int')
+        mocked_metadata['PartitionKey'] = meta.Datum('sample_part', 'string')
         mocked_metadata['Label'] = meta.Datum(
             'Microsoft.Azure.ServiceBus', 'string'
         )
-        mocked_metadata['sys'] = meta.Datum(type='json', value='''
-            {
-                "MethodName": "ServiceBusSMany",
-                "UtcNow": "2020-06-18T05:39:12.2860411Z",
-                "RandGuid": "bb38deae-cc75-49f2-89f5-96ec6eb857db"
+        mocked_metadata['MessageReceiver'] = meta.Datum(type='json', value='''
```

Comment:
This is the new section of metadata received?

Reply:
Yes, this is the new section of metadata provided by the extension. We still support the old **mocked_metadata['sys']** in line 192.